Pcap++/src/DpdkDevice.cpp

```diff
@@ -346,7 +348,11 @@ namespace pcpp
 	bool DpdkDevice::initQueues(uint8_t numOfRxQueuesToInit, uint8_t numOfTxQueuesToInit)
 	{
 		rte_eth_dev_info devInfo;
-		rte_eth_dev_info_get(m_Id, &devInfo);
+		if (rte_eth_dev_info_get(m_Id, &devInfo) < 0)
```

Comment:
In recent DPDK 24.11 they added [`__rte_warn_unused_result`](https://github.com/DPDK/dpdk/blob/ca1690ebd224f148268285b15b97441ccdbdd07e/lib/ethdev/rte_ethdev.h#L3538) to `rte_eth_dev_info_get` and `rte_eth_link_get` so we have to consider the return value

Reply:
maybe add PCPP_LOG_ERROR no ?

Reply:
Yes I agree, update in this commit: https://github.com/seladb/PcapPlusPlus/pull/1774/commits/6a44d0885da211c613acd7bba9f81ff62cc19dc3